app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/KeyboardWidget.java

```diff
@@ -1049,7 +1049,7 @@ private void handleText(String aText, boolean skipCase) {
             postInputCommand(() -> connection.commitText(text, 1));
         }
 
-        if (!mIsCapsLock && mCurrentKeyboard.getAlphabeticCapKeyboard() == null) {
```

Comment:
We can't remove `&& mCurrentKeyboard.getAlphabeticCapKeyboard() == null`. It will cause when switching to the Cap mode, typing any character on the keyboard, it will force to switch back to the un-cap mode.

Reply:
It seems to be acceptable in iOS keyboard. Let me check it with an Android phone.